targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F429xI/device/TOOLCHAIN_IAR/stm32f429xx_flash.icf

```diff
@@ -17,7 +17,7 @@ define symbol __ICFEDIT_region_CCMRAM_end__   = 0x1000FFFF;
 /*-Sizes-*/
 /*Heap 1/2 of ram and stack 1/8*/
 define symbol __ICFEDIT_size_cstack__ = 0x6000;
-define symbol __ICFEDIT_size_heap__   = 0x18000;
```

Comment:
Preferably just remove those `/*Heap 1/2 of ram and stack 1/8*/` and just tell the numbers, like `/* 64kB of HEAP */`